src/Symfony/Bundle/TwigBundle/RuntimeExtensionInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle;
+
+/**
+ * Provide a marker to automatically add the "twig.runtime" tag to the
+ * associated service.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+interface RuntimeExtensionInterface
```

Comment:
but then it means that your library providing a reusable extension can either decide to couple itself to Symfony, or cannot benefit from autowiring when using it with Symfony (and then, other frameworks supporting autowiring also need to add their own version of this interface).

So I'm in favor of adding the marker interface in Twig.